examples/0_basic/7_revoke_vc.rs

```diff
@@ -0,0 +1,192 @@
+    StrongholdSecretManager::builder()
+      .password("secure_password_2")
+      .build(random_stronghold_path())?,
+  );
+  let (_, alice_document, _): (Address, IotaDocument, KeyPair) = create_did(&client, &mut secret_manager_alice).await?;
+
+  // Create a new empty revocation bitmap. No credential is revoked yet.
+  let revocation_bitmap: RevocationBitmap = RevocationBitmap::new();
+
+  // Add the revocation bitmap to the DID document of the issuer as a service.
+  let service: IotaService = Service::from_json_value(json!({
+    "id": issuer_document.id().to_url().join("#my-revocation-service")?,
+    "type": RevocationBitmap::TYPE,
+    "serviceEndpoint": revocation_bitmap.to_endpoint()?
+  }))?;
+  issuer_document.insert_service(service);
+
+  // Resolve the latest output and update it with the given document.
+  let alias_output: AliasOutput = client.update_did_output(issuer_document.clone()).await?;
+
+  // Because the size of the DID document increased, we have to increase the allocated storage deposit.
+  // This increases the deposit amount to the new minimum.
+  let rent_structure: RentStructure = client.get_rent_structure()?;
+  let alias_output: AliasOutput = AliasOutputBuilder::from(&alias_output)
+    .with_minimum_storage_deposit(rent_structure)
+    .finish(client.get_token_supply()?)?;
+
+  // Publish the updated Alias Output.
+  issuer_document = client.publish_did_output(&secret_manager_issuer, alias_output).await?;
+
+  // Create a credential subject indicating the degree earned by Alice.
+  let subject: Subject = Subject::from_json_value(json!({
+    "id": alice_document.id().as_str(),
+    "name": "Alice",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts",
+    },
+    "GPA": "4.0",
+  }))?;
+
+  // Create an unsigned `UniversityDegree` credential for Alice.
+  // The issuer also chooses a unique `RevocationBitmap` index to be able to revoke it later.
+  let service_url = issuer_document.id().to_url().join("#my-revocation-service")?;
+  let credential_index: u32 = 5;
+  let status: Status = RevocationBitmapStatus::new(service_url, credential_index).into();
+
+  // Build credential using subject above, status, and issuer.
+  let mut credential: Credential = CredentialBuilder::default()
+    .id(Url::parse("https://example.edu/credentials/3732")?)
+    .issuer(Url::parse(issuer_document.id().as_str())?)
+    .type_("UniversityDegreeCredential")
+    .status(status)
+    .subject(subject)
+    .build()?;
+
+  // Sign the Credential with the issuer's verification method.
+  issuer_document.sign_data(&mut credential, key_pair.private(), "#key-1", ProofOptions::default())?;
+  println!("Credential JSON > {:#}", credential);
+
+  // ===========================================================================
+  // Revocation of the Verifiable Credential.
+  // ===========================================================================
+
```

Comment:
Again maybe we should demonstrate that the credential can be successfully verified at this point before it gets revoked?